composer.json

```diff
@@ -0,0 +1,24 @@
+{
+    "name": "govcms/govcms8_uikit_starter",
+    "description": "Provides uikit starter theme to govCMS8",
+    "type": "drupal-theme",
+    "license": "GPL-2.0+",
+    "require": {
+        "drupal/core": "8.6.10"

```

Comment:
do we need a hard requirement on drupal/core version - this would mean that it will need to be re-released for every core patch?

Reply:
I will refactor this soon. This was more of a test-bed. @tobybellwood